packages/plugin-page-view-tracking/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { PageViewTrackingPlugin } from './page-view-tracking';
+export * from './typings/page-view-tracking';
```

Comment:
Should we make type exposure similar to other packages?

Reply:
We will need it when we are going to build `analytics-ma-browser`.  The type will be used for the init options.

Reply:
not saying we shouldn't expose. asking if we should make it similar to https://github.com/amplitude/Amplitude-TypeScript/blob/3909e7fcac0429eb047a9cf04ee96c36fc85e573/packages/analytics-browser/src/index.ts#L27

Reply:
Oh, I see what you means. Will change. Thanks.